tests/test_resource_upload.py

```diff
@@ -0,0 +1,165 @@
+import pytest
+import botocore.stub
+import boto3
+import os
+import json
+from mbtools import copy_resources_s3
+
+
+test_dir = 'test_content/'
+f1 = 'test_content/example1.txt'
+f2 = 'test_content/example2.mp4'
+f3 = 'test_content/example3.jpeg'
+
+f1_data = {"txt_data": 2}
+f2_data = {"mp4_data": "video"}
+f3_data = {"jpeg_data": "picture"}
+
+metadata_file = "metadata/tags.json"
+
+metadata_tags = [{"mime_type": "video/mp4",
+                  "original_filename": "Example_2.mp4",
+                  "s3_key": 'contents/raise/resources/f83a0cabe1fc9bfe35d709' +
+                            '47e9c0ed8e5285b76e',
+                  "sha1": "f83a0cabe1fc9bfe35d70947e9c0ed8e5285b76e"
+                  },
+                 {"mime_type": "video/mp4",
+                  "sha1": "d3614ca5260104f401a96415b448943725c2feb0",
+                  "original_filename": "TwoLines.mp4",
+                  "s3_key": "contents/raise/resources/d3614ca5260104f401a964" +
+                  "15b448943725c2feb0"}
+                 ]
+
+s3_response = {'Contents':
+               [
+                {'Key': 'contents/raise/resources/51ca61528ae211e79e1f9ad0b' +
```

Comment:
Minor: There's inconsistent use of `'` and `"` in some of this code that would be nice to cleanup